### ISSUE TITLE
g16.optinput does not report what 'OPT' is added

### ISSUE THREAD

**Issue by polyluxus** [polyluxus/tools-for-g16.bash#41]
For example:
```
INFO    : Configuration file '[...]/.g16.toolsrc' applied.
WARNING : Link0 directive '%NProcShared=24' found.
WARNING : The statement will be replaced by script values.
WARNING : Link0 directive '%Mem=20000MB' found.
WARNING : The statement will be replaced by script values.
INFO    : Removed comment: Automagically created with g16.submit.sh
INFO    : Removed comment: [...]/bin/g16.submit -p24 -m20000 df-b97d3tzvp.irc.fwd.com
INFO    : Removed keyword 'IRC(RCFC,forward,MaxPoints=25,StepSize=10,MaxCycle=100)'.
INFO    : Removed keyword 'guess(read)'.
INFO    : Added 'guess(read)' to the route section.
INFO    : Removed keyword 'geom(check)'.
INFO    : Added 'geom(check)' to the route section.
INFO    : Checkpoint file 'df-b97d3tzvp.irc.fwd.opt.chk' does not exists and will be created.
INFO    : Removed keyword 'MaxDisk=30000MB'.
INFO    : Added 'MaxDisk=30000MB' to the route section.
INFO    : Written modified inputfile 'df-b97d3tzvp.irc.fwd.opt.com'.
INFO    : g16.optinput.sh is part of tools-for-g16.bash 0.0.17 (2018-10-09)
```

The line where it states what `OPT` is added is missing.

**Comment by polyluxus** [comment]
also valid for irc

**Comment by polyluxus** [comment]
This was actually fixed in commit f9dc4c4a5959c40f9b4330ee25b2b8c76a687940 (I forgot to reference it there).

**Comment by polyluxus** [comment]
```
<USER>@cluster:~/ [...] $ g16.prepare -l0 -jtest start.xyz
INFO    : Configuration file '[...]/.g16.toolsrc' applied.
INFO    : Applied route section:
INFO    : # PM6
INFO    : Setting charge (0) and multiplicity (1).
INFO    : Using caption 'Calculation: test; chrg 0; mult 1; uhf 0; 2019-02-04 22:09:40 (CET)'.
INFO    : Checkpoint file 'test.chk' does not exists and will be created.
INFO    : Added 'MaxDisk=30000MB' to the route section.
INFO    : Written modified inputfile 'test.com'.
INFO    : g16.prepare.sh is part of tools-for-g16.bash 0.1.0 (2019-02-04)
<USER>@cluster:~/ [...] $ g16.optinput -otight test.com
INFO    : Configuration file '[...]/.g16.toolsrc' applied.
WARNING : Link0 directive '%NProcShared=12' found.
WARNING : The statement will be replaced by script values.
WARNING : Link0 directive '%Mem=2000MB' found.
WARNING : The statement will be replaced by script values.
INFO    : Removed comment: Automagically created with g16.prepare.sh
INFO    : Removed comment: /home/<USER>/bin/g16.prepare -l0 -jtest start.xyz
INFO    : Added 'guess(read)' to the route section.
INFO    : Added 'geom(check)' to the route section.
INFO    : Added 'OPT(tight)' to the route section.                             ( --- !!! --- )  
WARNING : Checkpoint file 'test.chk' does not exist.
WARNING : Gaussian will likely fail to run this calculation.
INFO    : Checkpoint file 'test.opt.chk' does not exists and will be created.
INFO    : Removed keyword 'MaxDisk=30000MB'.
INFO    : Added 'MaxDisk=30000MB' to the route section.
INFO    : Written modified inputfile 'test.opt.com'.
INFO    : g16.optinput.sh is part of tools-for-g16.bash 0.1.0 (2019-02-04)
```
Works as expected see `( --- !!! --- )` mark.
